lib/haml_lint/linter/alt_text.rb

```diff
@@ -0,0 +1,11 @@
+module HamlLint
+  class Linter::AltText < Linter
+    include LinterRegistry
+
+    def visit_tag(node)
+      if node.tag_name == 'img' && !node.has_hash_attribute?(:alt)
```

Comment:
@sds the only tag that explicitly requires the attribute is `img`. `area` tags require them when using an `href`, and `applet` tags don't need them if they have fallback content in the tag body.